setup.py

```diff
@@ -1,10 +1,18 @@
 import platform
+import shlex
 from glob import glob
-from os import environ
+from os import environ, pathsep
 
 from setuptools import Extension, setup
 
-dconv_source_files = glob("./deps/double-conversion/double-conversion/*.cc")
+dconv_includes = environ.get(
+    "UJSON_BUILD_DC_INCLUDES",
+    "./deps/double-conversion/double-conversion",
+).split(pathsep)
```

Comment:
This would be `[""]` if `UJSON_BUILD_DC_INCLUDES` is not set. I'm not sure if it's likely to do any damage but I'd rather the current working directory isn't added to include path. Can you make sure that empty strings are removed.

Reply:
> This would be `[""]` if `UJSON_BUILD_DC_INCLUDES` is not set.

To be pedantic, it would be `["./deps/double-conversion/double-conversion"]` if the variable were not set, and `[""]` only if the variable were set to the empty string.

I can easily filter out empty strings, though; anyone who wants to add the current working directory as a relative path for some reason could still use `.`.